pkg/sdk/testint/accounts_integration_test.go

```diff
@@ -1,291 +1,696 @@
+		require.Len(t, policies, 0)
+		require.NoError(t, err)
+	}
+
+	t.Run("set / unset resource monitor", func(t *testing.T) {
+		resourceMonitor, resourceMonitorCleanup := testClientHelper().ResourceMonitor.CreateResourceMonitor(t)
 		t.Cleanup(resourceMonitorCleanup)
-		opts := &sdk.AlterAccountOptions{
+
+		require.Nil(t, resourceMonitor.Level)
+		err := client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
 			Set: &sdk.AccountSet{
-				ResourceMonitor: resourceMonitorTest.ID(),
+				ResourceMonitor: resourceMonitor.ID(),
 			},
-		}
-		err := client.Accounts.Alter(ctx, opts)
+		})
+		require.NoError(t, err)
+
+		resourceMonitor, err = testClientHelper().ResourceMonitor.Show(t, resourceMonitor.ID())
 		require.NoError(t, err)
+		require.NotNil(t, resourceMonitor.Level)
+		require.Equal(t, sdk.ResourceMonitorLevelAccount, *resourceMonitor.Level)
+
+		err = client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
+			Unset: &sdk.AccountUnset{
+				ResourceMonitor: sdk.Bool(true),
+			},
+		})
+		require.NoError(t, err)
+
+		resourceMonitor, err = testClientHelper().ResourceMonitor.Show(t, resourceMonitor.ID())
+		require.NoError(t, err)
+		require.Nil(t, resourceMonitor.Level)
 	})
 
-	t.Run("set and unset password policy", func(t *testing.T) {
-		passwordPolicyTest, passwordPolicyCleanup := testClientHelper().PasswordPolicy.CreatePasswordPolicy(t)
+	t.Run("set / unset policies", func(t *testing.T) {
+		authPolicy, authPolicyCleanup := testClientHelper().AuthenticationPolicy.Create(t)
+		t.Cleanup(authPolicyCleanup)
+		passwordPolicy, passwordPolicyCleanup := testClientHelper().PasswordPolicy.CreatePasswordPolicy(t)
 		t.Cleanup(passwordPolicyCleanup)
-		opts := &sdk.AlterAccountOptions{
+		sessionPolicy, sessionPolicyCleanup := testClientHelper().SessionPolicy.CreateSessionPolicy(t)
+		t.Cleanup(sessionPolicyCleanup)
+		packagesPolicyId, packagesPolicyCleanup := testClientHelper().PackagesPolicy.Create(t)
+		t.Cleanup(packagesPolicyCleanup)
+
+		err := client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
 			Set: &sdk.AccountSet{
-				PasswordPolicy: passwordPolicyTest.ID(),
+				PackagesPolicy: packagesPolicyId,
 			},
-		}
-		err := client.Accounts.Alter(ctx, opts)
+		})
 		require.NoError(t, err)
 
-		// now unset
-		opts = &sdk.AlterAccountOptions{
-			Unset: &sdk.AccountUnset{
-				PasswordPolicy: sdk.Bool(true),
+		err = client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
+			Set: &sdk.AccountSet{
+				PasswordPolicy: passwordPolicy.ID(),
 			},
-		}
-		err = client.Accounts.Alter(ctx, opts)
+		})
 		require.NoError(t, err)
-	})
 
-	t.Run("set and unset session policy", func(t *testing.T) {
-		sessionPolicyTest, sessionPolicyCleanup := testClientHelper().SessionPolicy.CreateSessionPolicy(t)
-		t.Cleanup(sessionPolicyCleanup)
+		err = client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
```

Comment:
How about deferring unsetting policies in the test cleanup?

Reply:
Done